api/osrd_infra/models/train_schedule.py

```diff
@@ -22,9 +33,6 @@ class TrainScheduleModel(models.Model):
     )
     speed_limit_composition = models.CharField(max_length=128, null=True)
     options = models.JSONField(null=True, blank=True, validators=[PydanticValidator(TrainScheduleOptions)])
-
-    # Simulation outputs
-    mrsp = models.JSONField(validators=[PydanticValidator(MRSP)])
-    base_simulation = models.JSONField()
-    eco_simulation = models.JSONField(null=True)
-    modes_and_profiles = models.JSONField(default=list)
+    trainscheduleoutput = models.OneToOneField(
```

Comment:
Why not 
```suggestion
    train_schedule_output = models.OneToOneField(
```
?

Reply:
Indeed looks like in our current code we are considering python convetion > sql convention, I will change that